src/components/ButtonPanel.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Button from './Button';
+
+class ButtonPanel extends React.Component {
+  // eslint-disable-next-line class-methods-use-this
+  renderButton(i) {
+    return <Button name={i} />;
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
+          {this.renderButton('AC')}
```

Comment:
Hi, the render function isn't doing anything serious, you could just render the component multiple time direct. But this is no deal breaker. 😁